js/viz/docs/docSankey.js

```diff
@@ -0,0 +1,484 @@
+            * @default false
+            */
+            visible: false,
+            /**
+            * @name dxSankeyOptions.links.border.width
+            * @type number
+            * @default 2
+            */
+            width: 1,
+            /**
+            * @name dxSankeyOptions.links.border.color
+            * @type string
+            * @default #000000
+            */
+            color: '#000000'
+        },
+        /**
+        * @name dxSankeyOptions.links.hoverStyle
+        * @type object
+        */
+        hoverStyle: {
+            /**
+            * @name dxSankeyOptions.links.hoverStyle.opacity
+            * @type number
+            * @default 0.5
+            */
+            opacity: 0.5,
+            /**
+            * @name dxSankeyOptions.links.hoverStyle.color
+            * @type number
+            * @default undefined
+            */
+            color: undefined,
+            /**
+            * @name dxSankeyOptions.links.hoverStyle.border
+            * @type object
+            */
+            border: {
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.border.visible
+                * @type boolean
+                * @default undefined
+                */
+                visible: undefined,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.border.width
+                * @type number
+                * @default undefined
+                */
+                width: undefined,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.border.color
+                * @type string
+                * @default undefined
+                */
+                color: undefined
+            },
+            /**
+            * @name dxSankeyOptions.links.hoverStyle.hatching
+            * @type object
+            */
+            hatching: {
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.direction
+                * @type Enums.HatchingDirection
+                * @default 'right'
+                */
+                direction: "right",
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.opacity
+                * @type number
+                * @default 0.75
+                */
+                opacity: 0.75,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.step
+                * @type number
+                * @default 6
+                */
+                step: 6,
+                /**
+                * @name dxSankeyOptions.links.hoverStyle.hatching.width
+                * @type number
+                * @default 2
+                */
+                width: 2,
+            }
+        },
+    },
+    /**
+    * @name dxSankeyOptions.onItemClick
+    * @extends Action
+    * @type function|string
+    * @type_function_param1 e:object
+    * @type_function_param1_field1 jQueryEvent:jQuery.Event:deprecated(event)
```

Comment:
There's no need to declare a deprecated field in a new widget. Also, the enumeration should start with 4 because the first three fields are described in the Action doccomment. This applies to all other event handlers.